circleci: failed tests (13465): TestReplicateRogueRemovedNode

The following test appears to have failed:

[#13465](https://circleci.com/gh/cockroachdb/cockroach/13465):

```
I160226 17:50:26.586580 raft/raft.go:472  [group 4] 1 became candidate at term 6
I160226 17:50:26.586614 raft/raft.go:521  [group 4] 1 received vote from 1 at term 6
I160226 17:50:26.586653 raft/raft.go:500  [group 4] 1 became leader at term 6
I160226 17:50:26.607938 client/db.go:488  failed batch: client/rpc_sender.go:83: failed to send RPC request Server.Batch: client is unhealthy
I160226 17:50:26.608235 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestMultiRangeEmptyAfterTruncate (0.16s)
	dist_sender_server_test.go:88: client/rpc_sender.go:83: failed to send RPC request Server.Batch: client is unhealthy
=== RUN   TestMultiRangeScanReverseScanDeleteResolve
I160226 17:50:26.611083 server/context.go:165  can't read available memory from cgroups (open /sys/fs/cgroup/memory/memory.limit_in_bytes: no such file or directory)
	setting default rocksdb cache size to 30 GiB (half of system memory)
I160226 17:50:26.615125 rpc/clock_offset.go:154  monitoring cluster offset
I160226 17:50:26.617467 server/node.go:352  store store=0:0 ([]=) not bootstrapped
I160226 17:50:26.619312 raft/raft.go:459  [group 1] 1 became follower at term 5
I160226 17:50:26.619382 raft/raft.go:235  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I160226 17:50:26.619418 raft/raft.go:539  [group 1] 1 is starting a new election at term 5
I160226 17:50:26.619477 raft/raft.go:472  [group 1] 1 became candidate at term 6
--
W160226 17:50:38.362522 kv/dist_sender.go:621  failed to invoke BeginTransaction ["a",/Min), Put ["a",/Min), Put ["b",/Min): txn "kv/dist_sender_server_test.go:586 TestPropagateTxnOnPushError" id=cc4d176f key="a" rw=true pri=0.01225872 iso=SERIALIZABLE stat=PENDING epo=1 ts=1456509038.359889032,0 orig=1456509038.359889032,0 max=1456509038.359889032,0 failed to push "kv/dist_sender_server_test.go:560 TestPropagateTxnOnPushError.func1" id=55cf3b10 key="b" rw=false pri=0.00546374 iso=SERIALIZABLE stat=PENDING epo=0 ts=1456509038.355049999,0 orig=1456509038.355049999,0 max=1456509038.355049999,0
I160226 17:50:38.362932 client/db.go:488  failed batch: txn "kv/dist_sender_server_test.go:586 TestPropagateTxnOnPushError" id=cc4d176f key="a" rw=true pri=0.01225872 iso=SERIALIZABLE stat=PENDING epo=2 ts=1456509038.359889032,0 orig=1456509038.359889032,0 max=1456509038.409889032,0 failed to push "kv/dist_sender_server_test.go:560 TestPropagateTxnOnPushError.func1" id=55cf3b10 key="b" rw=false pri=0.00546374 iso=SERIALIZABLE stat=PENDING epo=0 ts=1456509038.355049999,0 orig=1456509038.355049999,0 max=1456509038.355049999,0
I160226 17:50:38.417728 storage/replica_command.go:457  auto-gc'ed 55cf3b10 (1 intents)
I160226 17:50:38.421375 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestPropagateTxnOnPushError (0.19s)
FAIL
FAIL	github.com/cockroachdb/cockroach/kv	32.757s
=== RUN   TestCombinable
--- PASS: TestCombinable (0.00s)
=== RUN   TestBatchSplit
--- PASS: TestBatchSplit (0.00s)
=== RUN   TestFlagsToStr
--- PASS: TestFlagsToStr (0.00s)
=== RUN   TestBatchRequestGetArg
--- PASS: TestBatchRequestGetArg (0.00s)
=== RUN   TestKeyNext
--- PASS: TestKeyNext (0.00s)
--
E160226 17:51:44.486131 storage/client_test.go:734  engine 1: missing key "a"
E160226 17:51:44.486191 storage/client_test.go:734  engine 2: missing key "a"
I160226 17:51:45.488337 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
I160226 17:51:45.488499 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
I160226 17:51:45.488570 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- FAIL: TestReplicateRogueRemovedNode (3.39s)
	testing.go:169: storage/client_raft_test.go:1328: condition failed to evaluate within 3s: storage/client_test.go:752: expected [16 0 5], got [16 0 0]
=== RUN   TestReplicateReAddAfterDown
E160226 17:51:45.526928 gossip/gossip.go:212  different node IDs were set for the same gossip instance (2147483647, 1)
I160226 17:51:45.527170 raft/raft.go:459  [group 1] 1 became follower at term 5
I160226 17:51:45.527265 raft/raft.go:235  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I160226 17:51:45.527302 raft/raft.go:539  [group 1] 1 is starting a new election at term 5
I160226 17:51:45.527337 raft/raft.go:472  [group 1] 1 became candidate at term 6
I160226 17:51:45.527368 raft/raft.go:521  [group 1] 1 received vote from 1 at term 6
I160226 17:51:45.527432 raft/raft.go:500  [group 1] 1 became leader at term 6
W160226 17:51:45.527762 gossip/gossip.go:456  bad address from gossip node {1 {localhost 1} }: gossip/resolver/resolver.go:95: unknown address network "localhost" for 1
--
I160226 17:51:53.770673 <autogenerated>:5  grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:52337: getsockopt: connection refused"; Reconnecting to "127.0.0.1:52337"
I160226 17:51:53.770704 <autogenerated>:5  grpc: Conn.transportMonitor exits due to: grpc: the client connection is closing
--- PASS: TestInOrderDelivery (0.54s)
=== RUN   Example_rebalancing
--- PASS: Example_rebalancing (0.08s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	22.490s
=== RUN   TestBatchBasics
I160226 17:51:44.683624 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.00s)
=== RUN   TestBatchGet
I160226 17:51:44.684546 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.00s)
=== RUN   TestBatchMerge
I160226 17:51:44.687718 storage/engine/rocksdb.go:137  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.00s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

--- a/storage/client_raft_test.go
+++ b/storage/client_raft_test.go
@@ -1347,6 +1347,8 @@ func TestReplicateRogueRemovedNode(t *testing.T) {
 	// lease will cause GC to do a consistent range lookup, where it
 	// will see that the range has been moved and delete the old
 	// replica.
+	mtc.manualClock.Increment(int64(storage.DefaultLeaderLeaseDuration+
+		storage.ReplicaGCQueueInactivityThreshold) + 1)
 	mtc.stores[2].ForceReplicaGCScanAndProcess()
 	mtc.waitForValues(roachpb.Key("a"), []int64{16, 0, 0})
 

